bundles/EcommerceFrameworkBundle/Controller/AdminOrderController.php

```diff
@@ -178,19 +185,27 @@ public function listAction(Request $request)
             'paginator' => $paginator,
             'pimcoreUser' => \Pimcore\Tool\Admin::getCurrentUser(),
             'listPricingRule' => new \Pimcore\Bundle\EcommerceFrameworkBundle\PricingManager\Rule\Listing(),
-            'defaultCurrency' => \Pimcore\Bundle\EcommerceFrameworkBundle\Factory::getInstance()->getEnvironment()->getDefaultCurrency(),
-            'formatter' => \Pimcore::getContainer()->get('pimcore.locale.intl_formatter')
+            'defaultCurrency' => Factory::getInstance()->getEnvironment()->getDefaultCurrency(),
+            'formatter' => $formatter,
```

Comment:
trailing comma

Reply:
What's the problem? Is it against the code style?

Reply:
It is a good. Also symfony do this:
https://symfony.com/doc/current/contributing/code/standards.html#structure
`Add a comma after each array item in a multi-line array, even after the last one;`

So there is no change in the line above in diffs if someone add a line.

Reply:
I was just concerned about neat code and was not aware that it has a benefit. Good to know that Symfony also recommends it 😉 thanks!